src/meta/client/tests/it/grpc_client.rs

```diff
@@ -43,18 +46,48 @@ async fn test_grpc_client_action_timeout() {
     assert_eq!(got, expect);
 }
 
-#[ignore]
 #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
 async fn test_grpc_client_handshake_timeout() {
     let srv_addr = start_grpc_server();
 
-    let timeout = Duration::from_secs(2);
-    let res =
-        MetaGrpcClient::try_create(vec![srv_addr], "", "", Some(timeout), None, None).unwrap();
+    // timeout will happen
+    {
+        let timeout = Duration::from_secs(2);
```

Comment:
i cannot catch the point: why timeout 2s will timeout, and timeout 3s will success?

Reply:
https://github.com/datafuselabs/databend/pull/8036/files#diff-6e99c87bc772e9ac0a583e62ab5f4bec7bbdd00e87232eef7b620456505be0f1R33-R34

Reply:
I add some comments now:)